bitnami/postgresql/templates/primary/statefulset.yaml

```diff
@@ -532,9 +532,8 @@ spec:
             {{- end }}
           {{- end }}
           env:
-            {{- $database := required "In order to enable metrics you need to specify a database (.Values.auth.database or .Values.global.postgresql.auth.database)" (include "postgresql.v1.database" .) }}
             - name: DATA_SOURCE_URI
-              value: {{ printf "127.0.0.1:%d/%s?sslmode=disable" (int (include "postgresql.v1.service.port" .)) $database }}
+              value: {{ printf "127.0.0.1:%d/?sslmode=disable" (int (include "postgresql.v1.service.port" .)) }}
```

Comment:
Could you check also the read statefulset ? That file has the same piece of code:
https://github.com/bitnami/charts/blob/08e8c5859d21cd76feeba0474a545c35680b163e/bitnami/postgresql/templates/read/statefulset.yaml#L435-L437

Reply:
Looks like the same is on the read replicas one, committing now. Thanks!